extra/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ MYSQL_ADD_EXECUTABLE(replace replace.c COMPONENT Client)
 TARGET_LINK_LIBRARIES(replace mysys)
 
 IF(UNIX)
-  MYSQL_ADD_EXECUTABLE(resolve_stack_dump resolve_stack_dump.c)
+  MYSQL_ADD_EXECUTABLE(resolve_stack_dump resolve_stack_dump.c COMPONENT Client)
```

Comment:
Why?

Reply:
a) reduce Deb/RPM differences, Debian had this in client.
b) its man page was installed in client.

Reply:
`resolve_stack_dump` is a server tool and it used to be in mariadb-server deb package too (up to 10.5, iirc).
But now many server tools are in the client package already, and both rpm and deb server packages require client anyway.
So ok, let's keep it in the client